modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json

```diff
@@ -120,12 +120,12 @@
 		"description": "Grant access to wireless status display",
 		"read": {
 			"ubus": {
-				"iwinfo": [ "assoclist" ]
+				"iwinfo": [ "assoclist", "wps_get_status" ]
```

Comment:
Shouldn't that be `"hostapd.*": [ "wps_status" ]` ?

Reply:
Yes you are right.